cli/ballerina-cli/src/main/resources/cli-help/ballerina-graphql.help

```diff
@@ -1,27 +1,40 @@
 NAME
-       ballerina-graphql - Generate Ballerina client sources from a GraphQL config file 
-                        configured with GraphQL schemas(SDL) and GraphQL queries.
+       ballerina-graphql - Generate Ballerina client sources for a GraphQL config file
+       and generate GraphQL schema for a Ballerina GraphQL service.
 
 
 SYNOPSIS
-       bal graphql [-i | --input] <graphql-configuration-file-path> [-o | --output] <output-location> 
+       bal graphql [-i | --input] <graphql-configuration-file-path>
+                   [-o | --output] <output-location>
+       bal graphql [-i | --input] <ballerina-graphql-service-file-path>
+                   [-o | --output] <output-location>
+                   [-s | --service] <service-base-path>
 
 
 DESCRIPTION
-       Generate the Ballerina GraphQL client sources for a given GraphQL config file 
-       configured with GraphQL schemas(SDL) and GraphQL queries.
+       Generate the Ballerina GraphQL client sources for a given GraphQL config file configured
+       with GraphQL schemas specified by GraphQL Schema Definition Language(SDL) and GraphQL
+       queries or export a GraphQL schema(SDL) for a given Ballerina GraphQL service.
     
-       The generated Ballerina sources will be written into the provided output location.
+       The generated Ballerina sources or GraphQL schema files will be written into the provided
+       output location.
 
 
 OPTIONS
-        -i, --input <path>
-           Path to a GraphQL config file configured with GraphQL schemas(SDL) and GraphQL queries.
-
-        -o, --output <path>
-           Location of the generated Ballerina source code. If this path is not specified,
-           the output will be written to the same directory from which the command is run.
-
+        -i, --input <graphql-configuration-file-path | ballerina-graphql-service-file-path>
+            This is mandatory input. The given GraphQL config file input configured with GraphQL
```

Comment:
```suggestion
            This is mandatory input. The given GraphQL config file input is configured with GraphQL
```

Reply:
This doesn't convey the exact message. Hence changed as follows. `The given GraphQL config file which is configured with GraphQL schemas (SDL) and queries, will generate the Ballerina GraphQL client sources.`